subprojects/arrow.wrap

```diff
@@ -0,0 +1,10 @@
+[wrap-file]
+directory = apache-arrow-20.0.0
+source_url = https://dlcdn.apache.org/arrow/arrow-20.0.0/apache-arrow-20.0.0.tar.gz
```

Comment:
Could you use https://www.apache.org/dyn/closer.lua URL for all Apache Software Foundation projects?

See also: https://infra.apache.org/release-download-pages.html

```suggestion
source_url = https://www.apache.org/dyn/closer.lua?action=download&filename=arrow/arrow-20.0.0/apache-arrow-20.0.0.tar.gz
```